pgjdbc/src/main/java/org/postgresql/Driver.java

```diff
@@ -62,7 +62,7 @@
  */
 public class Driver implements java.sql.Driver {
 
-  protected static @Nullable Driver registeredDriver;
+  private static @Nullable Driver registeredDriver;
```

Comment:
Why?

Reply:
The AWS driver extends from this driver and the variable is shared. So during the test when we call a `.deregister()` from AWS driver, the community driver might be the one thats deregistered but an AWS driver would still be registered. (When running all tests, this causes a problem)